### PR TITLE
Remove other init scripts and improve init script comments

### PR DESCRIPTION
**Pull request by udpatil**
## Describe your changes and provide context
Keep the relevant init script and add some comments + fail fast in script if error

## Testing performed to validate your change
verified that the script works

### CODE REVIEW

File: scripts/initialize_local_chain.sh [hunk 1]
```diff
@@ -37,13 +26,15 @@ jq '.validators[0] += {"pub_key":'$KEY'}' ~/.sei/config/tmp_genesis_2.json > ~/.
 mv ~/.sei/config/tmp_genesis_3.json ~/.sei/config/genesis.json && rm ~/.sei/config/tmp_genesis.json && rm ~/.sei/config/tmp_genesis_2.json
 
 echo "Creating Accounts"
-python3  loadtest/scripts/populate_genesis_accounts.py 50 loc
+# create 10 test accounts + fund them
+python3  loadtest/scripts/populate_genesis_accounts.py 10 loc
 
 ~/go/bin/seid collect-gentxs
+# update some params in genesis file for easier use of the chain localls (make gov props faster)
 cat ~/.sei/config/genesis.json | jq '.app_state["gov"]["deposit_params"]["max_deposit_period"]="300s"' > ~/.sei/config/tmp_genesis.json && mv ~/.sei/config/tmp_genesis.json ~/.sei/config/genesis.json
```
Comment (BrandonWeng) [inline scripts/initialize_local_chain.sh:48]: Do we need this? Can we just use the default

File: scripts/initialize_local_chain.sh [hunk 2]
```diff
@@ -37,13 +26,15 @@ jq '.validators[0] += {"pub_key":'$KEY'}' ~/.sei/config/tmp_genesis_2.json > ~/.
...
 cat ~/.sei/config/genesis.json | jq --arg release_date "$(date +"%Y-%m-%d")" '.app_state["mint"]["params"]["token_release_schedule"]=[{"date": $release_date, "token_release_amount": "999999999999"}]' > ~/.sei/config/tmp_genesis.json && mv ~/.sei/config/tmp_genesis.json ~/.sei/config/genesis.json
-cat ~/.sei/config/genesis.json | jq '.app_state["gov"]["voting_params"]["expedited_voting_period"]="4s"' > ~/.sei/config/tmp_genesis.json && mv ~/.sei/config/tmp_genesis.json ~/.sei/config/genesis.json
+cat ~/.sei/config/genesis.json | jq '.app_state["gov"]["voting_params"]["expedited_voting_period"]="10s"' > ~/.sei/config/tmp_genesis.json && mv ~/.sei/config/tmp_genesis.json ~/.sei/config/genesis.json
 cat ~/.sei/config/genesis.json | jq '.app_state["oracle"]["params"]["vote_period"]="1"' > ~/.sei/config/tmp_genesis.json && mv ~/.sei/config/tmp_genesis.json ~/.sei/config/genesis.json
 cat ~/.sei/config/genesis.json | jq '.app_state["distribution"]["params"]["community_tax"]="0.000000000000000000"' > ~/.sei/config/tmp_genesis.json && mv ~/.sei/config/tmp_genesis.json ~/.sei/config/genesis.json
```
Comment (BrandonWeng) [inline scripts/initialize_local_chain.sh:53]: can we remove any unnecessary overrides

**Review by BrandonWeng — APPROVED** [review]
lgtm with some nits

File: scripts/initialize_local_chain.sh [hunk 3]
```diff
@@ -37,13 +26,15 @@ jq '.validators[0] += {"pub_key":'$KEY'}' ~/.sei/config/tmp_genesis_2.json > ~/.
...
+cat ~/.sei/config/genesis.json | jq '.app_state["gov"]["voting_params"]["voting_period"]="30s"' > ~/.sei/config/tmp_genesis.json && mv ~/.sei/config/tmp_genesis.json ~/.sei/config/genesis.json
 cat ~/.sei/config/genesis.json | jq --arg release_date "$(date +"%Y-%m-%d")" '.app_state["mint"]["params"]["token_release_schedule"]=[{"date": $release_date, "token_release_amount": "999999999999"}]' > ~/.sei/config/tmp_genesis.json && mv ~/.sei/config/tmp_genesis.json ~/.sei/config/genesis.json
-cat ~/.sei/config/genesis.json | jq '.app_state["gov"]["voting_params"]["expedited_voting_period"]="4s"' > ~/.sei/config/tmp_genesis.json && mv ~/.sei/config/tmp_genesis.json ~/.sei/config/genesis.json
+cat ~/.sei/config/genesis.json | jq '.app_state["gov"]["voting_params"]["expedited_voting_period"]="10s"' > ~/.sei/config/tmp_genesis.json && mv ~/.sei/config/tmp_genesis.json ~/.sei/config/genesis.json
 cat ~/.sei/config/genesis.json | jq '.app_state["oracle"]["params"]["vote_period"]="1"' > ~/.sei/config/tmp_genesis.json && mv ~/.sei/config/tmp_genesis.json ~/.sei/config/genesis.json
```
Comment (BrandonWeng) [inline scripts/initialize_local_chain.sh:52]: Can we just set this to 1 in the code?

Comment (udpatil) [reply scripts/initialize_local_chain.sh:52]: 1s?  isnt that too fast to submit + vote yes?

File: scripts/initialize_local_chain.sh [hunk 4]
```diff
@@ -1,33 +1,22 @@
-
-
 #!/bin/bash
+# require success for commands
+set -e
 
+# set key name
 keyname=admin
-#docker stop jaeger
```
Comment (philipsu522) [inline scripts/initialize_local_chain.sh:6]: might be nice to leave this

**Review by philipsu522 — APPROVED** [review]

File: scripts/initialize_local_chain.sh (hunk 1 above)
Comment (udpatil) [reply scripts/initialize_local_chain.sh:48]: The default for gov is 2 days for normal and 1 day for expedited, which is too long for local chain

File: scripts/initialize_local_chain.sh (hunk 2 above)
Comment (udpatil) [reply scripts/initialize_local_chain.sh:53]: which ones are unnecessary?

File: scripts/initialize_local_chain.sh (hunk 4 above)
Comment (udpatil) [reply scripts/initialize_local_chain.sh:6]: sg, i havent recently uncommented to use this but could be useful to leave for the future if someone needs to use

File: scripts/initialize_local_chain.sh (hunk 3 above)
Comment (BrandonWeng) [reply scripts/initialize_local_chain.sh:52]: Oh im referring to "oracle"]["params"]["vote_period"]` i just updated it here https://github.com/sei-protocol/sei-chain/pull/717/files#diff-4f014297fd4b20647b36d9d0187633ac132726e7984acfabc3bc30d391b87371R28 

Default was 10

**Pull request merged**